templates/vue/src/components/tyde/TydeStage.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <div class="module-container" :style="nodeStyles">
+    <div class="stage-content">
+      <h1 class="tyde-stage-title">{{ node.title }}</h1>
+      <section class="button-container">
+        <button
+          v-for="question in questions"
+          :key="question.id"
+          class="tyde-stage-button"
+          @click="openLightbox(question.id)"
+        >
+          <div class="tyde-stage-button-image-container">
+            <img class="tyde-stage-button-image" :src="question.imageURL" />
+          </div>
+          <p class="tyde-stage-button-text">{{ question.title }}</p>
+        </button>
+      </section>
+      <footer class="tyde-stage-footer">
+        <button v-if="done" class="tyde-button-next" @click="$emit('next')">
+          Next
+        </button>
+      </footer>
+    </div>
+    <lightbox v-if="isLightboxOpen" :node-id="lightboxId" @close="closeLightbox" />
+  </div>
+</template>
+
+<script>
+import { mapGetters } from "vuex"
+import Lightbox from "../Lightbox"
+import Helpers from "../../utils/Helpers"
+
+import StageBackground from "../../assets/stage-bg.png"
```

Comment:
This background should come from the thumbnail of the stage node, not hard coded

Reply:
Thanks, fixed.